Sources/Logging/Logging.swift

```diff
@@ -392,8 +400,21 @@ extension Logger {
 }
 
 extension Logger.Level: Comparable {
+    private var severityOrdering: Int {
```

Comment:
this will need to become `@inlinable`

Reply:
Does it? Or just `@usableFromInline var ...`?

I am probably misunderstanding what the compiler can and can't do - but I assume that the protocol conformance methods are not inlinable, but code within it would be?